CHANGELOG.md

```diff
@@ -245,7 +249,7 @@
 
 #### :rocket: New Feature
 
-- Experimental support of tagged template literals, e.g. ```sql`select * from ${table}```. https://github.com/rescript-lang/rescript-compiler/pull/6250
+- Experimental support of tagged template literals, e.g. `` sql`select * from ${table} ``. https://github.com/rescript-lang/rescript-compiler/pull/6250
```

Comment:
Can you undo the formatting changes and just keep the entry for the new feature?